node_modules/oae-principals/lib/model.userArchive.js

```diff
@@ -0,0 +1,33 @@
+/*!
+ * Copyright 2017 Apereo Foundation (AF) Licensed under the
+ * Educational Community License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License. You may
+ * obtain a copy of the License at
+ *
+ *     http://opensource.org/licenses/ECL-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an "AS IS"
+ * BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+var _ = require('underscore');
+var util = require('util');
+
+var AuthzUtil = require('oae-authz/lib/util');
+var TenantsAPI = require('oae-tenants');
+
+/**
+ * The UserArchive model.
+ *
+ * @param  {String}     aliasTenant                 The alias of the tenant this user belongs to
+ * @param  {String}     idArchive                   The globally unique id for this userArchive. e.g.: g:cam:math-101
+ */
+module.exports.UserArchive = function(aliasTenant, idArchive) {
```

Comment:
`ArchiveUser` would be a better name; `UserArchive` implies that the item in question is an archive *of* users, instead of an archived user.